### PR TITLE
Added caregiver award md document

### PR DESCRIPTION
**Pull request by SiminaB**

### CODE REVIEW

**Comment by lpantano** [comment]
This is great, thank you!

two questions:

1-when I check the deploy preview I see this is coming from an old version of the web page. Can you make sure you sync the files on the master file of this repo to this branch?
2-we need to add a link to this new page, maybe here is an option: http://bioc2020.bioconductor.org/call-for-abstracts

thanks!

File: docs/caregiver-awards.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Travel or other expenses related to the attendee's participation in the meeting
+- Food
+- Tickets to local attractions
+
+### Selection of awards
+
+- If the number of requests for grants exceeds the funding, preference will be given to 
+applicants in earlier career stages.
+
+### Application misrepresentation
```
Comment (mtmorgan) [inline docs/caregiver-awards.md]: I'm not sure that this is necessary; it leaves a kind of sour and distrustful note, and I'm not sure that we would act on any 'suspicious' claim anyway.

Comment (SiminaB) [reply docs/caregiver-awards.md]: Thanks, I just took that out!

**Comment by SiminaB** [comment]
> This is great, thank you!
> 
> two questions:
> 
> 1-when I check the deploy preview I see this is coming from an old version of the web page. Can you make sure you sync the files on the master file of this repo to this branch?
> 2-we need to add a link to this new page, maybe here is an option: http://bioc2020.bioconductor.org/call-for-abstracts
> 
> thanks!

Thanks @lpantano! I _think_ it should be synced now. I think we should link both this and the travel awards from the front page and highlight them, as right now the Travel Awards are a bit hard to find as well (bottom of http://bioc2020.bioconductor.org/travel-accommodations)

**Comment by lwaldron** [comment]
Can this be merged now @lpantano ?

**Comment by lpantano** [comment]
I need to create the links, will do today.

**Comment by lpantano** [comment]
merging this through: https://github.com/Bioconductor/BioC2020/pull/59

**Pull request closed without merging**